sunpy/net/helioviewer.py

```diff
@@ -20,53 +20,83 @@
 
 __all__ = ['HelioviewerClient']
 
+datasource_info = dict()
 
 class HelioviewerClient(object):
     """Helioviewer.org Client"""
-    def __init__(self, url="https://legacy.helioviewer.org/api/"):
+    def __init__(self, url="https://api.helioviewer.org/"):
         """
-        url : location of the Helioviewer API.  The default location points to
-            version 1 of the API.  Version 1 of the Helioviewer API is
-            currently planned to be supported until the end of April 2017.
+        url : location of the Helioviewer API. This url points to 
+        the Helioviewer API.
         """
         self._api = url
-
+        self.init_src_dict()
+    
+    def init_src_dict(self):
+        datasources = self.get_data_sources() # doctest: +REMOTE_DATA
+        global datasource_info
```

Comment:
You will need to redo this. We do not want to global variables.

Reply:
Okk